### DISCUSSION TITLE
Next 11.0.1, Image tag Loading slow

### DISCUSSION THREAD

**Discussion by Chanaka-Sandaruwan** [vercel/next.js#36700] [Help] [answered]
### Verify canary release

- [ ] I verified that the issue exists in Next.js canary release

### Provide environment information

 Operating System:
      Platform: Windows 
      Arch: x64
      Version: 21H2
    Binaries:
      Node: 16.13.2
      npm: 8.1.2
    Relevant packages:
      next: 11.0.1
      react: 17.0.2
      react-dom: 17.0.2

### What browser are you using? (if relevant)

Chrome Version 100.0.4896.127 (Official Build) (64-bit)

### How are you deploying your application? (if relevant)

next start

### Describe the Bug

The images load very slowly when I add the Image tag to my NextJS project, but work perfectly when I add the img tag. Please tell me how to use Image tags to load images more quickly.

![Screenshot 2022-05-03 164033](https://user-images.githubusercontent.com/104133044/166626004-8d90780b-13c7-4adf-9218-685ff74a9e51.png)

According to the above image, I receive high errors whenever I add the img tag, but Google Insights scores are quite high (89).

![Screenshot 2022-05-03 164919](https://user-images.githubusercontent.com/104133044/166626360-621fd594-9289-4d15-9113-fe1f62f7c0f2.png)

According to the above image, I receive low errors whenever I add the Image tag, but Google Insights scores are quite low (72).

Additionally, every time I add an Image tag, the images and page load more than 30 seconds, what should I do?

### Expected Behavior

In order to integrate images, I need to use the Image tag in a more user-friendly way.

### To Reproduce

There may be a problem with the Image tag on the Next 11, so we need to find another solution.

**Comment by balazsorban44** [comment, marked as answer]
Without a reproduction, it is hard to guess the problem, but the lower score seems to have multiple reasons judging from the screenshots.

For images specifically, the missing width and height attributes might be mistakenly reported: https://github.com/GoogleChrome/lighthouse/pull/13737#pullrequestreview-910640635

In any case, you might want to upgrade to Next.js 12 to leverage the latest changes to the Image component.

If you are using self-hosting, consider adding `sharp` as a dependency to improve performance as described in the documentation:

https://nextjs.org/docs/deployment#nodejs-server

**Comment by Chanaka-Sandaruwan** [comment]
Thank you, @balazsorban44, and
![Screenshot 2022-05-06 101821](https://user-images.githubusercontent.com/104133044/167068975-e03be44b-a9b4-4f60-9b6f-03d8e35e1781.png)

This is how I added all my images to my project and after upgrading to the latest version (12.1.6) and adding sharp, images are loading very quickly. However, when I set priority="true", all images will stop lazy loading, but when I remove priority="true", images start loading slowly again. Is there a way to deal with this?

**Reply by balazsorban44** [reply]
That is the purpose of `priority`:

> Lazy loading is automatically disabled for images using `priority`.

https://nextjs.org/docs/api-reference/next/image#priority

Consider only using it on images that need to be prioritized (ie: they are above the fold when loading the page.)

**Comment by Alireza17224** [comment]
setting 
```
priority="true"
```
will remove image lazy loading !!!
The speed will be higher than before but without any lazy loading!
Same Problem
